Sources/App/Views/Search/SearchShow+View.swift

```diff
@@ -28,7 +28,7 @@ extension SearchShow {
 
         override func pageTitle() -> String? {
             if model.query.count > 0 {
-                return "Search Results for &ldquo;\(model.query)&rdquo;"
+                return "Search Results for &ldquo;\(model.term)&rdquo;"
```

Comment:
Well spotted! 😍

Reply:
Think I caught all uses. Worth noting that this does a little funky in the scenario where there are no terms (only filters) but I don't think it's a major problem.

Reply:
Fixed in 007bdeb1c053248f4869feebfe47d299b33c61cc

Reply:
Aha fair enough, cheers Dave